docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -331,8 +363,14 @@ spec:
           service:
             metadata:
               annotations:
+ifeval::["{OpenStackPreviousInstaller}" == "director"]
```

Comment:
@pinikomarov Shouldn't this change to:` ifeval::["{OpenStackPreviousInstaller}" != "director_operator"] ` ?

Reply:
changed